lib/runner/concurrency/index.js

```diff
@@ -90,7 +179,7 @@ class Concurrency extends EventEmitter {
    * @param {Array} availColors
    * @return {Promise}
    */
-  runTestEnvironments(envs, args, availColors) {
+  runProcessTestEnvironment(envs, args, availColors) {
```

Comment:
We don't want to support this mode anymore: 
https://github.com/nightwatchjs/nightwatch/pull/3590#issuecomment-1450242970